nncf/torch/pruning/filter_pruning/algo.py

```diff
@@ -253,12 +256,36 @@ def _init_pruned_modules_params(self):
             self.pruning_quotas[cluster.id] = np.floor(self._modules_out_channels[cluster.elements[0].node_name] \
                                                        * self.pruning_quota)
 
+    def _calculate_output_shape(self, graph: NNCFGraph, node: NNCFNode) -> Tuple[int, ...]:
+        """
+        Calculates output shape of convolution layer by input edge.
+        :param graph: the model graph
```

Comment:
to be aligned with CodeStyle guidelines.

```suggestion
        Calculates output shape of convolution layer by input edge.
        
        :param graph: the model graph
```